### ISSUE TITLE
Warning: DatePickerAndroid has been merged with DatePickerIOS and will be removed in a future release.

### ISSUE THREAD

**Issue by maxencedouet** [GeekyAnts/vue-native-core#239]
**Description of the bug**
Warning about deprecated react native component using is printed on console when launching new repo following the installation tutorial. It is very annoying as it is on the same pipe than real errors.

**To Reproduce**
1. vue-native init my_project
2. npm install
3. npm run ios

**What I expected**
These warning should be fixed. Should we update the metro config or directly the vue-native-core module ?

**Screenshots**
![image](https://user-images.githubusercontent.com/42381786/76520117-93f37500-6462-11ea-9959-42745bcf209d.png)

**Did I use `vue-native-cli` to create the project?**
Yes

**Am I using Expo?**
Yes

**Development platform (please complete the following information):**
 - OS: MAC
 - Shell/terminal: ZSH
-  Expo-cli: 3.13.6

**The device on which I run my Vue Native app**
 - Device: iPhone8
 - OS: last

**Comment by RishabhKarnad** [comment]
@maxencedouet we'll prevent these deprecated modules from being automatically imported in the next release

**Comment by maxencedouet** [comment]
thk you very much for you rreactivity !
Do you know if we can subscribe to a newsletter to be informed about the release ? 
When do you plan it ?

**Comment by mjbriggs** [comment]
@RishabhKarnad I am planning on resolving issue #219 I would like to clarify that this pr is only responsible for removing deprecated dependencies and does not resolve the linked issue. Based on a quick perusal of the pr that seems to be the case, but I wanted to double check.

**Comment by RishabhKarnad** [comment]
@mjbriggs if you are referring to #249, it only removes the deprecated components to resolve this issue, but does not try to solve #219

**Comment by knot22** [comment]
Three warnings were appearing in my Vue Native projects in the Dev environment.  To suppress these the following code was added in the app.vue file:
```
import { LogBox } from "react-native";

created() {
    LogBox.ignoreLogs([
      'DatePickerIOS has been merged with DatePickerAndroid and will be removed in a future release.',
      'StatusBarIOS has been merged with StatusBar and will be removed in a future release.',
      'DatePickerAndroid has been merged with DatePickerIOS and will be removed in a future release.'
    ]);
  }
```

**Comment by RishabhKarnad** [comment]
Should be resolved by #288 in v0.2.0. Closing this issue.
